icssc/AntAlmanac
# Material UI Time Picker

## Summary
The previous, text entry time picker has been replaced by a material-ui pop-up. AM and PM options are also now supported. The minute hand is restricted to 60-minute increments, but it is still ignored when selecting the time.

![time-picker](https://user-images.githubusercontent.com/25188285/111806094-df6b0b00-888e-11eb-9474-e10d07af113d.gif)

## Test Plan
1. Try searching without specifying start/end times to make sure it functions normally first.
2. Enter a start time, check if results all come after specified time.
3. Enter an end time, check that all results come between the two times.
4. Click on the start time, then click clear, search, then make sure start time is no longer restricted, but that end time still is.
5. Clear end time and make sure classes span the entire day.

## Issues
#135
